tests/test_report_generator.py

```diff
@@ -13,16 +15,40 @@
 # cisagov Libraries
 from tpt_reports import report_generator
 
+FILE_PATH = "./tests/data"
+
 
 @pytest.fixture
-def test_dataframe():
-    """Define a fixture for a sample DataFrame."""
-    with open("tests/data/test.json", encoding="utf-8") as file:
+def test_payloads_dataframe():
+    """Define a fixture for the payloads DataFrame."""
+    with open(os.path.join(FILE_PATH, "test.json"), encoding="utf-8") as file:
         data = json.load(file)
         return pd.DataFrame.from_dict(data["payloads"])
 
 
-def test_format_table(test_dataframe):
+@pytest.fixture
+def test_dictionary():
+    """Define a fixture for the test json dictionary."""
```

Comment:
Use proper capitalization:
```suggestion
    """Define a fixture for the test JSON dictionary."""
```